### PR TITLE
Relay : added tests

### PR DESCRIPTION
**Pull request by v-Ajnava**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,4 +65,7 @@ src/build
 *.pubxml
 
 # [begoldsm] ignore virtual env if it exists.
-adlEnv/
\ No newline at end of file
+adlEnv/
+
+# real file
+/azure-sdk-testutils/devtools_testutils/mgmt_settings_real.py
```
Comment (lmazuel) [inline .gitignore]: Please remove this file from your PR, this is not necessary

File: azure-sdk-testutils/devtools_testutils/mgmt_settings_fake.py [hunk 2]
```diff
@@ -20,8 +20,7 @@
 CLIENT_OID = '00000000-0000-0000-0000-000000000000'
 # Cognitive Services tests
 CS_SUBSCRIPTION_KEY = '0000000000000000000000000000'
-# Event Grid key
-EVENT_GRID_KEY = '0000000000000000000000000000'
+
```
Comment (lmazuel) [inline azure-sdk-testutils/devtools_testutils/mgmt_settings_fake.py]: This is why the EventGrid tests are failing. You should not change this file at all in your PR

**Comment by lmazuel** [comment]
@v-Ajnava LGTM! Do you think the package is ready to be released?

**Comment by v-Ajnava** [comment]
@lmazuel yes, we can release the package.

**Pull request merged**